src/test/regress/expected/multi_extension.out

```diff
@@ -388,6 +388,28 @@ SELECT * FROM print_extension_changes();
                  | function worker_save_query_explain_analyze(text,jsonb) SETOF record
 (2 rows)
 
+-- Test upgrade paths for backported citus_pg_upgrade functions
+ALTER EXTENSION citus UPDATE TO '9.4-2';
+ALTER EXTENSION citus UPDATE TO '9.4-1';
+-- Should be empty result since upgrade+downgrade should be a no-op
```

Comment:
This comment is a lie. It's not a no-op in this case, since we explicitly keep the new version during the downgrade. The result from print_extension_changes is still empty, but that's simply because arguments don't change.

Reply:
makes sense, will address this subtlety in the comments